boot/bootdata/hivesft.inf

```diff
@@ -1,4 +1,4 @@
-[Version]
+﻿[Version]
```

Comment:
You probably (accidentally) added a BOM here.

Reply:
Yes, I used notepad. Fixed now, Thanks.